client/src/components/growth-book/growth-book-wrapper.tsx

```diff
@@ -77,17 +78,19 @@ const GrowthBookWrapper = ({
 
   useEffect(() => {
     async function setGrowthBookFeatures() {
-      if (!growthbookUri) return;
-
-      try {
-        const res = await fetch(growthbookUri);
-        const data = (await res.json()) as {
-          features: Record<string, FeatureDefinition>;
-        };
-        growthbook.setFeatures(data.features);
-      } catch (e) {
-        // TODO: report to sentry when it's enabled
-        console.error(e);
+      if (!growthbookUri) {
+        growthbook.setFeatures(defaultGrowthBookFeatures);
```

Comment:
yes, otherwise we should pass the uri env from github and include it in the sample.env (so contributors could load the landing page), and perhaps intercept the fetch call in the test, etc.

Reply:
Okay, that sounds good. Could you add a quick comment explaining that?